src/server/game/Maps/Map.h

```diff
@@ -728,14 +750,19 @@ class Map : public GridRefMgr<NGridType>
     void _ScriptProcessDoor(Object* source, Object* target, const ScriptInfo* scriptInfo) const;
     GameObject* _FindGameObject(WorldObject* pWorldObject, ObjectGuid::LowType guid) const;
 
+    time_t i_gridExpiry;
```

Comment:
can we use std::chrono?

Reply:
Hmm, I suppose but not sure what there is to gain from it 🤔

Reply:
readabiltiy

Reply:
Also it was decided a couple of years ago to move to std::chrono where possible

Reply:
Alright, if it is the prefered way I will have to look into it tomorrow - not sure if I can just swap it and expect everthing to work the same.

I've added/fixed the other things you suggested though.

Reply:
Alright, I've now converted time_t to std::chrono::seconds in the places where I saw fit. To my understanding time_t represents time in seconds hence why I also went with std::chrono::seconds.

In NGrid.h you'll find that ResetTimeTracker and UpdateTimeTracker still takes time_t as a parameter. That is becasue we use some TimeTracker which uses time_t and I thought it was a bit out of scope for this PR, to try change that too. I could still change the parameter for ResetTimeTracker and UpdateTimeTracker but then i'd also have to cast to std::chrono::seconds everywhere, where these methods are called, though its only a few places but ye...

Anyway, let me know what you think.